rotkehlchen/api/rest.py

```diff
@@ -4752,3 +4755,27 @@ def get_spam_assets_false_positives(self) -> Response:
             )
 
         return api_response(_wrap_in_ok_result(whitelisted_tokens), status_code=HTTPStatus.OK)
+
+    def add_to_spam(self, token: EvmToken) -> Response:
+        if token.protocol == SPAM_PROTOCOL:
+            return api_response(
+                result=wrap_in_fail_result('Token is marked as spam already'),
+                status_code=HTTPStatus.CONFLICT,
+            )
+
+        with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            toggle_token_spam_protocol(write_cursor=write_cursor, token=token, is_spam=True)
+
+        return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+
+    def remove_from_spam(self, token: EvmToken) -> Response:
```

Comment:
Same as above

1. Rename
2. Add 1liner docstring

```suggestion
    def remove_token_from_spam(self, token: EvmToken) -> Response:
```